pkgs/os-specific/linux/musl/default.nix

```diff
@@ -61,11 +61,12 @@ stdenv.mkDerivation rec {
     configureFlagsArray+=("--syslibdir=$out/lib")
   '';
 
+  CFLAGS="-fstack-protector-strong" + lib.optionalString hostPlatform.isPower " -mlong-double-64";
```

Comment:
This isn't the same as only setting CFLAGS during configure-- have you checked the output is identical on x86_64? Hopefully doesn't matter but better safe than sorry....

Reply:
And just curious about why this flag is needed, but I haven't read through the discussion/your website yet so apologies if all this has been addressed.

Reply:
The rationale here was that since CFLAGS might contain a space now, it might show up as two options, one of which ./configure will not understand

Reply:
The original problem is that long double defaults to 128-bits on ppc64le (even without `--with-long-double-128` which is required for glibc interestingly) but musl wants 64-bit long double.